doc/manual/manual.tex

```diff
@@ -7610,10 +7610,10 @@ \subsubsection{Using reconstructed surface velocities by GPlates}
 section~\ref{parameters:Boundary_20velocity_20model}. Possible options include the data directory
 and file name of the velocity file/files, the time step (in model units, mostly seconds or years depending on the 
 ``\texttt{Use years in output instead of seconds}'' flag) and the points that define the 2D plane.
-The parameter ``\texttt{Interpolation width}'' is used to smooth the provided velocity files by
-a moving average filter. All velocity data points within this distance are averaged to determine the
-actual boundary velocity at a certain mesh point. This parameter is usually set to 0 (no interpolation, use
-nearest velocity point data) and is only needed in case the original setting is unstable or slowly converging.
+%The parameter ``\texttt{Interpolation width}'' is used to smooth the provided velocity files by
```

Comment:
It is not a good idea to comment out text like this. If it is no longer relevant, we should delete it. I don't understand why this parameter got removed. Maybe @gassmoeller remembers?

Reply:
Yes we should delete this whole sentence. The parameter was removed a while ago when GPlates implemented this ability (so users can now select in GPlates whether they want their velocities to be smoothened. I apparently forgot to update the manual when the parameter was removed.

Reply:
I understand, I have removed the sentence.